src/components/inputs/FileUploader.tsx

```diff
@@ -0,0 +1,152 @@
+import { CloseRounded, FileUploadRounded, GridViewRounded, ListRounded } from '@mui/icons-material';
+import { Box, Grid, IconButton, List, ListItem, ListItemIcon, ListItemText, Paper, Stack, Typography } from '@mui/material';
+import { makeStyles } from 'makeStyles';
+import { useCallback, useEffect, useRef, useState } from 'react';
+import { useDropzone } from 'react-dropzone';
+
+const useStyles = makeStyles()((theme) => ({
+  fileUploader: {
+    display: 'block',
+  },
+  uploadBox: {
+    padding: theme.spacing(2),
+    borderRadius: `${theme.shape.borderRadius}px`,
+    border: '2px solid white',
+    '&:hover': {
+      backgroundColor: theme.palette.primary.main,
+      opacity: [0.9, 0.8, 0.7],
+    },
+  },
+}));
+
+type GridFileItemProps = {
+  index: number;
+  deleteFile: (index: number) => void;
+  file: File;
+};
+
+const GridFileItem = ({ index, deleteFile, file }: GridFileItemProps) => {
+  const ImageURL = useRef(URL.createObjectURL(file));
```

Comment:
```suggestion
  const imageURL = useRef(URL.createObjectURL(file));
```

Det er vanligvis kun classes og komponenter som har stor forbokstav